pvlib/irradiance.py

```diff
@@ -790,6 +791,10 @@ def haydavies(surface_tilt, surface_azimuth, dhi, dni, dni_extra,
         projection. Must supply ``solar_zenith`` and ``solar_azimuth``
         or supply ``projection_ratio``.
 
+    return_components: bool (optional, default=False)
```

Comment:
```suggestion
    return_components : bool, default False
```
I know how you originally did it matches `perez`, but this is the style we use nowadays (it is on my to-do list to go through and update old docstrings to this format)